src/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (CMAKE_CXX_COMPILER_LAUNCHER)
         string(LENGTH "" SOURCE_PATH_SIZE)
     endif()
 endif()
+if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
+    set(SOURCE_PATH_SIZE 3)
```

Comment:
this needs explanation. SOURCE_PATH_SIZE could, in general, get a comment explaining the value it contains.

Reply:
Right, this is still WIP. Only for checking it with CI.

Reply:
Sorry, missed the WIP part.